internal/service/dms/replication_task.go

```diff
@@ -315,3 +360,60 @@ func dmsReplicationTaskRemoveReadOnlySettings(settings string) (*string, error)
 	cleanedSettingsString := string(cleanedSettings)
 	return &cleanedSettingsString, nil
 }
+
+func dmsStartReplicationTask(id string, conn *dms.DatabaseMigrationService) error {
```

Comment:
Nit. These 2 new functions don't need to start with `dms` as they are in the `internal/service/dms` package.
Existing functions that do start with a service name prefix are hangovers from the pre-service packages days.